docs/source/app_developers_guide/docker.rst

```diff
@@ -0,0 +1,416 @@
+**************************
+Using Sawtooth with Docker
+**************************
+
+
+This document guides you through the process of setting up Hyperledger Sawtooth
+for application development using Docker Compose, and then introduces basic
+Sawtooth concepts necessary for application development.
+
+After following the steps in this document, you will be able to perform the
+following tasks:
+
+* run Sawtooth
+* submit transactions
+* view blocks, transactions and state with the CLI
+* start and stop validators and transaction processors
```

Comment:
This document walks through the process of setting up Hyperledger Sawtooth
for application development using Docker Compose, introduces some of the basic 
Sawtooth concepts necessary for application development, and walks through
performing the following tasks:
* Submit transactions to the REST API
* View blocks, transactions, and state with the `sawtooth` CLI tool
* Start and stop validators and transaction processors